app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java

```diff
@@ -300,7 +315,7 @@ public void onResume() {
             int bmWidth = _bmpCurrentImage.getWidth();
             int bmHeight = _bmpCurrentImage.getHeight();
             _columns = bmWidth < 90 ? bmWidth : 45;
-            _rows = _columns * bmHeight / bmWidth;
+            _rows = (int)Math.round(_columns * 1.0 * bmHeight / bmWidth);
```

Comment:
Where is space?
And why is there mathematic round here?

Reply:
In the exact meaning I, too, is probably better to use mathematical rounding
That is, I have exact 45 33, and in the range goes 45 34. So that, I've decided use round in both